inc/agent.class.php

```diff
@@ -296,6 +314,15 @@ public function handleAgent($metadata) {
          'itemtype'     => $metadata['itemtype'] ?? 'Computer'
       ];
 
+      $input['ip_version'] = 4;
+      if (isset($_SERVER['REMOTE_ADDR']) && filter_var($_SERVER['REMOTE_ADDR'], FILTER_VALIDATE_IP, FILTER_FLAG_IPV6)) {
+         $input['ip_version'] = 6;
+      }
+
+      if (isset($_SERVER['REMOTE_ADDR'])) {
+         $input['ip_binary'] = inet_pton($_SERVER['REMOTE_ADDR']);
```

Comment:
You should probably handle the case `inet_pton()` returns `false` even if this should not happen.